ietf/templates/meeting/group_proceedings.html

```diff
@@ -64,6 +76,12 @@ <h1>{{ entry.group }}</h1>
               </a>
               <br>
           {% endfor %}
+            {% if entry.session.video_stream_url %}
+                <a href="https://www.meetecho.com/ietf{{ meeting.number }}/recordings#{{ entry.group.acronym.upper }}">
```

Comment:
This probably needs a helper along the lines of `session_recording_url` that hands the template the appropriate URL if there is one. The appropriate URL will almost surely vary over time if it doesn't already.

Reply:
This is using the same pattern that several other places in the codebase use. If it's getting it wrong for the URL varying over time, it's doing that in production already.

It _would_ be good, however, to have all of them use a single helper function so the meetecho url isn't repeated through the codebase. It would be even better to move the url pattern into settings.